src/grpc/infer_handler.h

```diff
@@ -664,11 +665,56 @@ class InferHandlerState {
 
     bool ReceivedNotification() { return received_notification_; }
 
+    // Returns true ONLY when GRPC_ERROR from CORE is waiting to be processed.
+    bool IsGRPCError()
```

Comment:
This function name feels a little unintuitive that is has a side effect of changing the `grpc_stream_error_state_`.